template/src/pageComponents/buy-me-coffee/Profile.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import UserAvatar from './Avatar.svg';
+
+export default function Profile() {
+  // TODO: Fetch these values dynamically
+  const coffeeReceivedCount = 90;
+  const userName = '0xPudgy';
+  const aboutUser = 'I am a content creator producing videos about NFTs!';
+
+  return (
+    <section className="font-inter flex flex-col justify-start gap-8 md:flex-row md:items-center">
+      {/* Fetch user avatar dynamically */}
+      <Image alt="User Avatar" src={UserAvatar} className="h-28 w-28" />
+      <div className="flex flex-col items-start justify-start gap-2">
+        <h1 className="w-96 text-3xl font-semibold text-white">{userName}</h1>
+        <p className="text-base font-normal leading-normal text-zinc-400">{aboutUser}</p>
+        <p className="flex items-center justify-start gap-2">
+          <Image
+            alt="Coffees Received"
+            src="/coffee.svg"
```

Comment:
@ppiyush13 should this be an import instead of a string path?

Reply:
@arsood I have placed the `coffee.svg` under a folder `public` in the root directory, which can be directly referenced in the code starting with `/`.

https://nextjs.org/docs/pages/building-your-application/optimizing/static-assets